HARK/ConsumptionSaving/ConsPortfolioModel.py

```diff
@@ -244,9 +242,6 @@ def _calcwFunc(AdjustPrb, AdjustCount, ShareNowCount, vFunc_adj, CRRA):
             evVals = AdjustPrb*vFunc_adj[0][ShareIndex](evalgrid) + (1-AdjustPrb)*vFunc_adj[1][ShareIndex](evalgrid)
             with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):
                 evValsNvrs = utility_inv(evVals,gam=CRRA)
-            evVals = np.insert(evVals, 0, 0.0)
-            evValsNvrs = np.insert(evVals, 0, 0.0)
-
```

Comment:
Yeah, this was exactly where the error was popping in from. If there was a divide by zero error which was ignored by `np.errstate` there was a mismatch between the length of `evVals` and `evValsNvrs`